### PR TITLE
Adding the DUNS table and data loader.

### PR DESCRIPTION
**Pull request by klrBAH**
Adding the DUNS table and data loader.

Technical Release Notes:
- Added DUNS table, migration (plus migrations for dev merge)
- Added LoadDUNS.py to load DUNS files from sftp site or locally. Requires "sam_duns" section in Broker section of config.

### CODE REVIEW

**Comment by dpb-bah** [comment]
The codeclimate issues are valid and noted, though the cyclomatic complexity issues are simply due to the benchmark checks.

File: dataactcore/migrations/versions/c048f1dcdfa2_.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+"""empty message
```
Comment (alburde1) [inline dataactcore/migrations/versions/c048f1dcdfa2_.py]: Need a meaningful message. We usually put `Merge 9649eb55fa0b and 967fbc28a6ef ` for example. This file should also be renamed with `merge_9649e_and_967fb` tacked onto the end of the existing name.

File: dataactcore/migrations/versions/9649eb55fa0b_add_duns_index.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+"""add_duns_index
```
Comment (alburde1) [inline dataactcore/migrations/versions/9649eb55fa0b_add_duns_index.py]: Why is this underscored? `Add DUNS index` seems more human-readable

File: dataactcore/migrations/versions/ff4728a82180_add_duns_table.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
+"""add_duns_table
```
Comment (alburde1) [inline dataactcore/migrations/versions/ff4728a82180_add_duns_table.py]: Why is this underscored? `Add DUNS table` seems more human-readable

File: dataactcore/scripts/loadDUNS.py [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+    daily = args.daily
+    benchmarks = args.benchmarks
+    update = args.update
+
+    with create_app().app_context():
+        configure_logging()
+        sess = GlobalDB.db().session
+
+        if monthly and daily:
+            print("For loading a single local file, you must provide either monthly or daily.")
```
Comment (alburde1) [inline dataactcore/scripts/loadDUNS.py]: We don't want print statements. The logger should be sufficient. (apply to all below print statements)

File: dataactcore/migrations/versions/ff4728a82180_add_duns_table.py [hunk 5]
```diff
@@ -33,7 +33,7 @@ def upgrade_data_broker():
     sa.Column('created_at', sa.DateTime(), nullable=True),
     sa.Column('updated_at', sa.DateTime(), nullable=True),
     sa.Column('duns_id', sa.Integer(), nullable=False),
-    sa.Column('awardee_or_recipient_uniqu', sa.Text(), nullable=True),
+    sa.Column('awardee_or_recipient_uniqu', sa.Text(), nullable=True, index=True),
```
Comment (alburde1) [inline dataactcore/migrations/versions/ff4728a82180_add_duns_table.py]: I think you still have to do the `op.create_index('ix_duns_awardee_or_recipient_uniqu', 'duns', ['awardee_or_recipient_uniqu'], unique=False)` thing. I'm not sure this works this way (we don't have it like this in any of our other migrations)

File: dataactcore/migrations/versions/c048f1dcdfa2_.py [hunk 6]
```diff
@@ -1,14 +1,14 @@
-"""empty message
+"""merge ff4728a82180 and 967fbc28a6ef
```
Comment (alburde1) [inline dataactcore/migrations/versions/c048f1dcdfa2_.py]: File needs renaming

**Pull request merged**